www/script.js

```diff
@@ -50,13 +50,14 @@ document.addEventListener('DOMContentLoaded', () => {
 	};
 	L.Map.include(RestoreViewMixin);
 
-	var map = L.map('map');
+	var map = L.map('map', { attributionControl: false });
 	if (init_l0_map.force || !map.restoreView()) {
 		map.setView(init_l0_map.center.split(', '), init_l0_map.zoom);
 	}
 
 	L.tileLayer('https://tile.openstreetmap.org/{z}/{x}/{y}.png',
-		{ attribution: 'Map &copy; <a href="https://www.openstreetmap.org">OpenStreetMap contributors</a>' }).addTo(map);
+		{ attribution: 'Map © <a href="https://www.openstreetmap.org">OpenStreetMap contributors</a>' }).addTo(map);
+	L.control.attribution({ prefix: '<a href="https://leafletjs.com/">Leaflet</a>' }).addTo(map);
```

Comment:
I don't think we need this Leaflet attribution. The map is pretty small as it is, and it's not commonly added.

Reply:
Makes sense. Removed now 
![image](https://github.com/Zverik/Level0/assets/3727288/500141a0-b4a2-45ef-b9f4-a63c6c30ac6f)